documentation/src/main/wiki/ScreenCollector.md

```diff
@@ -11,17 +11,17 @@ Screen Collector is responsible for collecting screenshot of the page or just pa
 
 Module name: **screen**
 
-**Note that you cannot maximize the window and specify the dimension at the same time. If no parameters provided, default browser size is set before taking screenshot.**
-
 ##### Parameters
 
 | Parameter | Value | Description | Mandatory |
 | --------- | ----- | ----------- | --------- |
 | `xpath` | xpath_to_element | Xpath to element(s) | optional (either xpath or css) |
 | `css` | css_selector_to_element | css selector to element(s)| optional (either xpath or css) |
+| `exclude-elements` | id or css selectors to elements separated by commas | If you provide elements in exclude-elements, they won't be checked by layout comparator, the difference won't be included for `pixelThreshold` and `percentageThreshold` and you won't see the difference on the mask. | no |
```

Comment:
`id or css selectors to elements separated by commas` - it's enough to say `CSS selector` - it supports comma-separated id or classes by default